src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -91,13 +98,37 @@ public TlsDetails create(final SSLEngine sslEngine) {
     }
 
     /**
-     * Close this client.
+     * Creates High Level Rest Client for SDK.
+     * @param hostAddress The address of OpenSearch cluster, client can connect to
+     * @param port The port of OpenSearch cluster
+     * @return The SDKClient implementation of RestHighLevelClient. The user is responsible for calling {@link #doCloseHighLevelClient()} when finished with the client
+     */
+    public RestHighLevelClient initializeRestClient(String hostAddress, int port) {
```

Comment:
Can we mark this as `@Deprecated` with a javadoc `@deprected Provided for compatibility with existing plugins to permit migration. New development should not use this client.`